vercel/resource_edge_config_item.go

```diff
@@ -0,0 +1,260 @@
+package vercel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/vercel/terraform-provider-vercel/client"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ resource.Resource              = &edgeConfigItemResource{}
+	_ resource.ResourceWithConfigure = &edgeConfigItemResource{}
```

Comment:
Is there a reason not to support Import, too?

Reply:
Sorry I had to rush out and forgot to come back and add a comment in my PR about this.

I think I need to write a function which has 4 parts for importing. If the edge config is scoped to the a team then the import ID would need `${edgeConfigId}-${teamId}-${key}-${value}`, unless I'm missing something?

Reply:
I think it'd need three, i'm not sure the reason for having `value` in the import, given it can read that from the API. 

Should just be `{teamId}/{edgeConfigId}/{key}`

Reply:
We have this pattern in other resources 🙏